filters/filters.go

```diff
@@ -38,10 +38,13 @@ type FilterContext interface {
 	// route.
 	Served() bool
```

Comment:
sure that we need this? maybe we can mark it deprecated, too, and until removed, use it only in the context of the old MarkServed() way? just wondering.